pkg/machines/components/vcpuModal.jsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import cockpit from 'cockpit';
+
+import { show_modal_dialog } from 'cockpit-components-dialog.jsx';
+import InfoRecord from './infoRecord.jsx';
+
+React;
+
+const _ = cockpit.gettext;
+
+const VCPUModalBody = ({ vcpus, cpu }) => (
+    <div className="modal-body">
+        <table className="vcpu-detail-modal-table">
+            <tr>
+                <td>
+                    <table className='form-table-ct'>
+                        <InfoRecord descr={_("Count")} value={vcpus.count}/>
+                        <InfoRecord descr={_("Placement")} value={vcpus.placement}/>
+                        <InfoRecord descr={_("Max")} value={vcpus.max}/>
+                    </table>
+                </td>
+                <td>
+                    <table className='form-table-ct'>
+                        <InfoRecord descr={_("Sockets")} value={cpu.sockets || 'N/A'}/>
+                        <InfoRecord descr={_("Threads")} value={cpu.threads || 'N/A'}/>
+                        <InfoRecord descr={_("Cores")} value={cpu.cores || 'N/A'}/>
```

Comment:
I agree with @jelkosz that this should be "1" rather than N/A if it actually is "1" (this is what I see in virt-manager at least)